packages/strapi-plugin-content-type-builder/controllers/validation/relations.js

```diff
@@ -5,10 +5,25 @@ const { validators, isValidName } = require('./common');
 const { typeKinds } = require('./constants');
 
 const REVERSE_RELATIONS = ['oneToOne', 'oneToMany', 'manyToOne', 'manyToMany'];
+const STRAPI_USER_RELATIONS = ['oneWay', 'manyWay'];
+
+const isValidNature = validNatures =>
+  function(value) {
+    const allowedRelations =
+      this.parent.target === 'strapi::user' ? STRAPI_USER_RELATIONS : validNatures;
```

Comment:
Suggestion: You could use a constant for `strapi::user` (either only for this file or globally). IMO, we should avoid hardcoding too much literal values in the code